oci-hooks/qm-device-manager/README.md

```diff
@@ -34,10 +44,14 @@ The device manager hook allows containers to request specific device access thro
    [Container]
    Annotation=org.containers.qm.device.audio=true
    Annotation=org.containers.qm.device.ttys=true
+   # Wayland seat support
+   Annotation=org.containers.qm.wayland.seat=seat0
    ```
 
 ### Podman Command Line
 
+#### Device Access
+
 ```bash
 # Run container with audio device access
 podman run --annotation org.containers.qm.device.audio=true qm
```

Comment:
I assume that one is confusing, it is for qm addition to device discovery
Please consider adding the generated command of 
/usr/lib/systemd/system-generators/podman-system-generator --dryrun 
And which output exist in the generated command of that part 
ExecStart= in quadlet